recipes/libgcc/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "4.8.5" %}
+
+package:
+  name: libgcc
+  version: {{ version }}
+
+requirements:
+  build:
+    - gcc {{ version }}
+  run:
+    - libgcc_s {{ version }}
+    - libgomp {{ version }}
+    - libgfortran {{ version }}
+    - libquadmath {{ version }}
+    - libstdc++ {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+test:
+  commands:
+    - true
```

Comment:
Thinking that we could test that all the other libraries are present, but that also creates an annoying maintenance burden. Feedback welcome.